compose-files/elastic-stack.yml

```diff
@@ -0,0 +1,69 @@
+
+  # Logging
+
+  # Aggregates logs and forwards them to Elasticsearch.
+  logstash:
+    image: docker.elastic.co/logstash/logstash-oss:${ELASTIC_VERSION:-7.6}
+    restart: always
+    container_name: logstash
+    hostname: logstash
+    expose:
+      - 12201/udp
+      - 5044
+    networks:
+      placeos:
+    volumes:
+      - ./config/logstash/config:/config
+      - ./config/logstash/patterns:/opt/logstash/extra_patterns
+    restart: always
+    command: logstash -f /config
+    << : *logging-env
+
+  # Run 'docker-compose run --rm validate-logstash-config' to quickly check the logstash config.
+  validate-logstash-config:
+    container_name: validate-logstash
+    image: docker.elastic.co/logstash/logstash-oss:${ELASTIC_VERSION:-7.6}
+    volumes:
+      - ./config/logstash/config:/config
```

Comment:
For relative paths, we need to use `${PWD}` instead

https://github.com/place-labs/partner-environment/pull/63

Reply:
ok fixed in https://github.com/place-labs/partner-environment/pull/49/commits/09d81dde4e88693299680d104d63ca7e8716515b